server/routes/api/ticket.js

```diff
@@ -109,6 +109,7 @@ router.post('/', async (req, res) => {
       'hasCharger',
       'notes',
     ]);
+    ticketInfo.UserId = req.user.id;
```

Comment:
We set the UserId based on who's authenticated... this prevents someone from "hacking" the system and creating a ticket for a different user.

Reply:
Oh yeah. I forgot that the UserId can be found in the req.